File: VB/IntAsEnumExample.Module.Win/Editors/MyEnumIntPropertyEditor.vb
Imports System
Imports DevExpress.ExpressApp.Editors
Imports DevExpress.ExpressApp.Model
Imports DevExpress.ExpressApp.Win.Editors
Imports IntAsEnumExample.Module.BusinessObjects

Namespace IntAsEnumExample.Module.Win.Editors

    <PropertyEditor(GetType(Integer), False)>
    Public Class MyEnumIntPropertyEditor
        Inherits EnumIntPropertyEditor(Of SampleEnum)

        Public Sub New(ByVal objectType As Type, ByVal model As IModelMemberViewItem)
            MyBase.New(objectType, model)
        End Sub
    End Class
End Namespace

File: CS/IntAsEnumExample.Module.Win/Editors/MyEnumIntPropertyEditor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp.Editors;
using DevExpress.ExpressApp.Model;
using DevExpress.ExpressApp.Win.Editors;
using IntAsEnumExample.Module.BusinessObjects;

namespace IntAsEnumExample.Module.Win.Editors {
    [PropertyEditor(typeof(int), false)]
    public class MyEnumIntPropertyEditor : EnumIntPropertyEditor<SampleEnum> {
        public MyEnumIntPropertyEditor(Type objectType, IModelMemberViewItem model)
            : base(objectType, model) {
        }
    }
}
